recipes-zarhus/packagegroups/packagegroup-zarhus.bb

```diff
@@ -24,3 +26,24 @@ RDEPENDS:${PN}-dbg = " \
     libgpiod-tools \
     devmem2 \
 "
+RDEPENDS:${PN}-alsa = " \
+    alsa-utils \
+    alsa-plugins \
+    alsa-lib \
+    alsa-tools \
+    alsa-state \
+    libasound \
+    alsa-utils-speakertest \
+    mpg123 \
+    gettext \
+    dialog \
+    ncurses \
```

Comment:
Is this needed? Usually recipes that need this to run should have this in RDEPENDS in their own recipe.